### PR TITLE
feat(alerts): Add descriptive header to alerts

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Before image

After 
image

I'm not 100% convinced we want to pull the chart into this frame though

### CODE REVIEW

File: src/sentry/static/sentry/app/views/alerts/details/chart.tsx [hunk 1]
```diff
@@ -69,77 +70,83 @@ type Props = {
   closed?: string;
 };
 
-export default class Chart extends React.PureComponent<Props> {
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/views/alerts/details/chart.tsx:72]: Just whitespace changes

**Review by scttcper — APPROVED** [review]

**Comment by billyvg** [comment]
@EvanPurkhiser can we fix the height on the content area to stretch to footer?

image

**Comment by EvanPurkhiser** [comment]
Ahh yes. Thanks @billyvg, I did notice this but I think I slipped on fixing it.

**Comment by EvanPurkhiser** [comment]
https://github.com/getsentry/sentry/pull/19055 @billyvg

**Pull request merged**